dap_aria_mapping/config/base.yaml

```diff
@@ -0,0 +1,164 @@
+SEMANTIC_TAXONOMY:
```

Comment:
there is a taxonomies.yaml file that this should probably go in rather than base.yaml